watson/cli.py

```diff
@@ -1013,13 +1013,30 @@ def frames(watson):
     for frame in watson.frames:
         click.echo(style('short_id', frame.id))
 
+def isTime(date_time_string):
+    match = re.search("^\d{1,2}:\d{1,2}$", date_time_string)
+    return match != None
+
+def isDateTime(date_time_string):
+    match = re.search("^\d{4}-\d{1,2}-\d{1,2}\s\d{1,2}:\d{1,2}$", date_time_string)
+    return match != None
+
+def getDateTimeToday(time_string):
+    date_today = arrow.now().floor("day")
+    return getMergedDateTime(date_today, time_string)
+
+def getMergedDateTime(date_time, time_string):
+    hours, minutes = time_string.split(":")
+    return date_time.shift(hours=int(hours), minutes=int(minutes))
```

Comment:
Those functions should probably go to the `utils` module (and be unit tested ;) ).

Reply:
Good point. 5729d3e71b